### PR TITLE
Smoke.mp4

### PR DESCRIPTION
**Pull request by KittenColony**
## About the PR
Adds several new cigarette brands in an attempt to shatter the monopoly Nomads and Interdyne have on the smokers market.

## Why / Balance
Nomads don't cut it anymore. The majority of people prefer Nomads for the fact that there isn't a better alternative.

## Technical details
adds a few .rsi files and some .yml files in the dV namespace

## Media
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:

add: Added Several new cigarette brands.

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/cartons.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - type: Sprite
+    sprite: DeltaV/Objects/Consumable/Smokeables/Cigarettes/Cartons/purple.rsi
+    layers:
+      - state: closed
+  - type: Item
+    sprite: DeltaV/Objects/Consumable/Smokeables/Cigarettes/Cartons/purple.rsi
+  - type: StorageFill
+    contents:
+      - id: CigPackPurple
+        amount: 5
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/cartons.yml:16]:
```suggestion
        amount: 5

```

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/cigarette.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - type: Clothing
+    sprite: Objects/Consumable/Smokeables/Cigarettes/cigarette.rsi
+    slots: [ mask ]
+  - type: SolutionContainerManager
+    solutions:
+      smokable:
+        maxVol: 10
+        reagents:
+          - ReagentId: Sugar
+            Quantity: 10
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/cigarette.yml:45]:
```suggestion
            Quantity: 10

```

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  description: It's sweetened!
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Consumable/Smokeables/Cigarettes/Packs/pink.rsi
+  - type: Item
+    sprite: DeltaV/Objects/Consumable/Smokeables/Cigarettes/Packs/pink.rsi
+  - type: StorageFill
+    contents:
+    - id: CigaretteCandy
+      amount: 5
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml]:
```suggestion
      amount: 5
      
```

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    sprite: DeltaV/Objects/Consumable/Smokeables/Cigarettes/Packs/purple.rsi
+  - type: Item
+    sprite: DeltaV/Objects/Consumable/Smokeables/Cigarettes/Packs/purple.rsi
+  - type: StorageFill
+    contents:
+    - id: CigaretteOlive
+      amount: 5
+
+- type: entity
+  id: CigPackCandy
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml:17]: Where's the carton of CandyCigPacks?

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml (hunk 3 above)
Comment (DangerRevolution) [reply Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml]: It's good form apparently to leave a line at the end.

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/cigarette.yml (hunk 2 above)
Comment (DangerRevolution) [reply Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/cigarette.yml:45]: It's good form apparently to leave a line at the end.

**Review by DangerRevolution — COMMENTED** [review]
CandyCigCarton pls

**Comment by KittenColony** [comment]
its far from done, I still have to do beef cigarettes and [redacted], I'm just trying to get my dev-environment to work

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml (hunk 4 above)
Comment (KittenColony) [reply Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml:17]: :3

**Comment by KittenColony** [comment]
fucking dev env wont work so im going to hope this currently works and try the other bits *later*

**Comment by KittenColony** [comment]
why the fuck

**Comment by KittenColony** [comment]
I give up

**Comment by KittenColony** [comment]
I NO LONGER GIVE UP!

**Comment by KittenColony** [comment]
I still need to add beef cigarettes and some other misc brands, as well as the vender contents but weh!

**Comment by KittenColony** [comment]
why do I smell burning?

File: Resources/Prototypes/Catalog/Fills/Crates/service.yml [hunk 5]
```diff
@@ -50,15 +50,20 @@
   - type: StorageFill
     contents:
       - id: CigCartonGreen
-        prob: 0.50
+        prob: 0.33
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/Fills/Crates/service.yml]:
```suggestion
        prob: 0.33 # DeltaV
```

File: Resources/Prototypes/Catalog/Fills/Crates/service.yml [hunk 6]
```diff
@@ -50,15 +50,20 @@
...
         orGroup: CigCarton1
       - id: CigCartonRed
         orGroup: CigCarton1
       - id: CigCartonBlue
-        prob: 0.50
+        prob: 0.33
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/Fills/Crates/service.yml]:
```suggestion
        prob: 0.33 # DeltaV
```

File: Resources/Prototypes/Catalog/Fills/Crates/service.yml [hunk 7]
```diff
@@ -50,15 +50,20 @@
...
         orGroup: CigCarton2
       - id: CigCartonBlack
         orGroup: CigCarton2
+      - id: CigCartonPurple # DeltaV
+        prob: 0.33
+        orGroup: CigCarton3 
+      - id: CigCartonCandy
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/Fills/Crates/service.yml]:
```suggestion
      - id: CigCartonCandy # DeltaV
```

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/cigs.yml [hunk 8]
```diff
@@ -5,6 +5,8 @@
     CigPackRed: 2
     CigPackBlue: 2
     CigPackBlack: 2
+    CigPackPurple: 2 # DeltaV 
+    CigPackCandy: 2 # Also DV
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/cigs.yml]:
```suggestion
    CigPackCandy: 2 # DeltaV
```

File: Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/randomitems.yml [hunk 9]
```diff
@@ -143,6 +143,8 @@
       - CigCartonRed
       - CigCartonBlue
       - CigCartonBlack
+      - CigCartonPurple
+      - CigCartonCandy
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/randomitems.yml]:
```suggestion
      - CigCartonPurple # DeltaV
      - CigCartonCandy # DeltaV
```

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

**Comment by KittenColony** [comment]
I should have put that in a long time ago

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Pull request merged**